doc/Learn/Mvux/WorkingWithRecords.md

```diff
@@ -0,0 +1,125 @@
+---
+uid: Uno.Extensions.Mvux.Records
+---
+
+# How to write Records with MVUX
+
+## What is a Record
+
+A [record](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record) behaves like a class*, offering the feature of **immutability**, where the values assigned to it remain unchanged once set. It's possible to create records using the `record` modifier, for example:
+
+```csharp
+public record MyRecord();
+```
+
+Records **can** be, but are not necessarily, immutable. Also, records introduce the `with` operator, which is a helpful tool to deal with immutable objects, we will see more about this operator in the **Updating records** section.
```

Comment:
> Records **can** be, but are not necessarily, immutable.

Here we should explain that we should use only the `init` instead of `set`

Note: I saw this is explain later ... maybe we can just move that sentence there then. The goal is to make clear that using the `set` makes a `record` **non** immutable and should be avoided! The sentence here alone only adds confusion IMO.

Reply:
Changes in next commit.